Init RO doc for transformation (#474)
keboola/connection-docs: transformations/mappings/index.md

![Table Clone](/transformations/mappings/table-clone.png)

The `CLONE` mapping will execute almost instantly for a table of any size (typically under 10 seconds) 
as it physically does not move any data.

> On the other hand, you can use [Read-Only Input Mapping](/transformations/mappings/#read-only-input-mapping) which makes available all buckets and tables with read access,

Reviewer: ```suggestion
On the other hand, you can use [Read-Only Input Mapping](/transformations/mappings/#read-only-input-mapping) which makes all buckets and tables available with read access,
```

> so there is no need to clone the tables into a new schema, you can simply read from these buckets and tables in the trace.

Reviewer: Co znamená to "trace"?

Author: Malo tam byt trasnformation :)

> This function is automatically enabled in the transformations.

Reviewer: ```suggestion
This function is automatically enabled in transformations.
```

##### Read-Only Input Mapping

> For using **Read-Only Input Mapping** there is no need to set anything else and for transformations, tables are automatically accessible in transformation.
> This also applies to linked buckets. Here you have to pay attention to the fact that buckets and tables belong
> to another project, so you have to access e.g. the database of another project in dependence on the backend.

Reviewer: ```suggestion
To use **Read-Only Input Mapping** there is no need to set anything. For transformations, all tables in Storage are automatically accessible in transformation.
This also applies to linked buckets. Pay attention to the fact that buckets and tables belong
to another project, so you need to access the tables using the fully qualified identifier including database and schema in the source project. 
```

{: .image-popup}

> ![Storage](/transformations/mappings/storage.png)

Reviewer: Tohle by měl být screenshot detailu, kde je vidět ID bucketu.

Author: Jo jasne :) pridal som.

> The way how it works is that you have read access to all tables that exist in the storage in your project.

Reviewer: ```suggestion
You have read access to all tables that exist in your project's Storage directly on the underlying backend. However, this means you need to use the internal ID of bucket (`in.c-main` instead of `main` as you see in the UI). 
```

{: .image-popup}
![Read Only Input Mapping in transformation](/transformations/mappings/read-only-transformation.png)

In the transformation (Snowflake) code I select from the table **"in.c-main"."users"** and create a new table  `create table "cities" as select "city" from "in.c-main"."users";`.

> Depending on the backend, access to tables is different, how to access individual tables depending on the backend is then explained directly in the documentation of individual backends.

Reviewer: ```suggestion
Depending on the backend, the SQL format is different. More info regarding access to individual tables depending on the backend can be found in the documentation of those individual backends.
```

Reviewer: Ale nevím, jestli do toho vůbec zabíhat...

Author: Asi by som to tam spomenul aby sme nejak upozrnili ze tam nieje nejaky jednotny sposob ako na to pristupovat ale v kazdom backende sa to selektuje inac. Potom v dokumentacii backedndu je na konci jak to ide selektnut.

Reviewer: Tak tam možná dát linky na jednotlivé ty backendy

Author: jasne. pridal som.

> This is all thanks to **Read-Only Input Mapping** which allows me to use the table created in the storage directly to read in the transformation.

Reviewer: ```suggestion
As you can see, **Read-Only Input Mapping** allows you to read the table created in the storage directly in the transformation.
```
